index.html

```diff
@@ -87,163 +85,165 @@ <h3 class="work-subtitle">Multi-Post Stories</h3>
-            <ul class="lang-list2">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
-          </div>
-          <div class="other-works">
-            <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
-            <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
-              been the industry's standard</p>
-            <ul class="lang-list2">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
-          </div>
-          <div class="other-works">
-            <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
-            <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
-              been the industry's standard</p>
-            <ul class="lang-list2">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
-          </div>
-          <div class="other-works">
-            <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
-            <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
-              been the industry's standard</p>
-            <ul class="lang-list2">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
-          </div>
-          <div class="other-works">
-            <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
-            <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
-              been the industry's standard</p>
-            <ul class="lang-list2">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
-          </div>
+      </div>
+      <div class="card">
+        <div class="other-works">
+          <h3 class="card-subtitle2">Profesional Art Printing Data</h3>
+          <p class="card-text2">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard</p>
+          <ul class="lang-list2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <div class="prj-link"><a class="prj-link-a" href="#">See project</a></div>
+        </div>
```

Comment:
- [ ] Great job with the cards, the hover effect is perfect 💯 Kindly try to rectify the height of the cards to match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213730819-c5ce6bab-9fde-4ca0-a333-0916b034594b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213730838-aeab4a56-469d-496e-b979-1f57ca342e18.png)